app/workers/articles/bust_cache_worker.rb

```diff
@@ -0,0 +1,13 @@
+module Articles
+  class BustCacheWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
+
+    def perform(article_id, cache_buster = "CacheBuster")
+      article = Article.find_by(id: article_id)
+
+      cache_buster.constantize.bust_article(article) if article
```

Comment:
I'm still not sure I like `constantize` but for the important purpose of not serializing class objects I second this :)